app/uk/gov/hmrc/gform/gform/DeclarationController.scala

```diff
@@ -249,7 +249,7 @@ class DeclarationController(
     for {
       _ <- gformBackEnd.updateUserData(form, maybeAccessCode)
     } yield Redirect(
-      routes.DeclarationController.showDeclaration(maybeAccessCode, cache.formTemplate._id, SuppressErrors.No)
+      routes.SummaryController.summaryById(cache.formTemplate._id, maybeAccessCode)
```

Comment:
Nope. What if there is genuine error on declaration page? We don't want to go to summary page in that case.